### PR TITLE
Update security and responsible disclosure policy

### PR DESCRIPTION
**Pull request by jkohrman**
Rewrite Spinnaker's public security policy to reflect the proposed changes described here: https://github.com/spinnaker/.github/pull/3

### CODE REVIEW

File: content/en/docs/community/security/_index.md [hunk 1]
```diff
@@ -6,16 +6,58 @@ description:
...
+
+#### Step 4: Mitigating Vulnerabilities
+
+Once a security issue has been confirmed and evaluated, the Security SIG will work with other members of the Spinnaker community to identify individuals to develop a patch or fix for each bug.  Code changes for security issues will be developed publicly unless otherwise accepted by majority vote of the Security SIG.  We will then track released security patches in the changelog for each release and update the CVE record.
+
+## I have another question not answered here.  Who should I talk to?
+
+For general questions about Spinnaker security, feel free to join us in the [Spinnaker Slack](http://join.spinnaker.io) [#security-sig](https://spinnakerteam.slack.com/archives/CFN8F5UR2) channel.  Questions or feedback regarding a security issue or vulnerability should be sent directly to [security@spinnaker.io](mailto://security@spinnaker.io).
+
+## I'd like to participatet in the [Security SIG](https://github.com/spinnaker/governance/tree/master/sig-security).  How do I join?
```
Comment (ezimanyi) [inline content/en/docs/community/security/_index.md:61]:
```suggestion
## I'd like to participate in the [Security SIG](https://github.com/spinnaker/governance/tree/master/sig-security).  How do I join?
```

**Review by ezimanyi — APPROVED** [review]

**Review by brian-armory — APPROVED** [review]
LGTM! Thanks for the PR.

**Comment by brian-armory** [comment]
@jkohrman FYI The repo this PR is made against is the repo for the new website that is a WIP. An additional PR will need to be made to https://github.com/spinnaker/spinnaker.github.io, which is the repo for the current site.

**Comment by brian-armory** [comment]
Opened this PR in the current active repo: https://github.com/spinnaker/spinnaker.github.io/pull/2107

**Pull request merged**